src/components/Table/Table.story.jsx

```diff
@@ -362,6 +362,33 @@ storiesOf('Watson IoT|Table', module)
       },
     }
   )
+  .add(
+    'Stateful Example with every third row unselectable',
+    () => (
+      <StatefulTable
```

Comment:
alignment of the checkboxes is off, as well as the left border of the row, is that intended?
![image](https://user-images.githubusercontent.com/3360588/66329925-03ce6400-e8f5-11e9-94e5-1bd55dee2d4a.png)